document/CoreFormat.mdk

```diff
@@ -775,26 +813,27 @@ A `result` object describes a single result produced by an analysis tool.
 
 ### `ruleId` property { #result-ruleId }
 
-A `result` object should contain a property named `ruleId` whose value is a string containing
+If the `level` property (&sect;[#result-level]) of a `result` object is anything other than `note`,
+then the `result` object shall contain a property named `ruleId` whose value is a string containing
```

Comment:
Fixed

---
In reply to: [61356581](https://github.com/sarif-standard/sarif-spec/pull/172#discussion_r61356581) [](ancestors = 61356581)